tests/personal_access_token_test.py

```diff
@@ -0,0 +1,101 @@
+import vcr
+from unittest import TestCase
+from contentful_management.personal_access_token import PersonalAccessToken
+from .test_helper import CLIENT, PLAYGROUND_SPACE
+
+PERSONAL_ACCESS_TOKEN_WITH_TOKEN = {
+    "sys": {
+        "type": "PersonalAccessToken",
+        "id": "exampletokenid",
+        "createdAt": "2015-05-18T11:29:46.809Z",
+        "updatedAt": "2015-05-18T11:29:46.809Z"
+    },
+    "name": "My Token",
+    "revokedAt": None,
+    "scopes": [
+        "content_management_manage"
+    ],
+    "token": "46d42a80f96db00393ffa867a753de126e658484dd8a20d209bcb7efcf3761b9"
```

Comment:
Sure, I can change that, but this is the one on our docs, so I figured it was safe.